Engine_Revit_UI/Convert/Architecture/FromRevit/Ceilings.cs

```diff
@@ -76,6 +76,8 @@ public static List<oM.Architecture.Elements.Ceiling> CeilingsFromRevit(this Ceil
                 newCeiling.SetCustomData(ceiling);
 
                 newCeiling.UpdateValues(settings, ceiling);
+
+                newCeiling.CustomData.Add("CeilingPattern", ceiling.CeilingPattern(settings, planarSurface));
                 ceilingList.Add(newCeiling);
             }
 
```

Comment:
I would recommend adding the following here to make the user aware that the information about ceiling openings has just been lost forever 🤣 

```
            if (disctionary.Values.Sum(x => x.Count) != 0)
            {
                BH.Engine.Reflection.Compute.RecordWarning("(...)");
            }
```